nss/integration-tests/helper_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/ubuntu/authd/internal/testutils"
 )
 
+const StatusNotFound = 2
```

Comment:
does it need to be exported? Shouldn’t it be in the test itself, before the test case struct?

Reply:
Yeah. This is a "leftover" of me trying to add more status codes to test them, but then realized that they are more related to the `getent` command rather than NSS itself. I'll update this.